docs/modules/clusters/pages/ucn-dynamic-client.adoc

```diff
@@ -0,0 +1,42 @@
+= Configure Client
+:description: Clients use the dynamic configuration API to update {ucn} on the cluster. 
+:page-enterprise: true
+:page-beta: false
+
+{description}
+
+You must create a _client.java_ file containing the configuration and save it to your configuration folder. 
+
+NOTE: Ensure that you have configured your data structure with the same {ucn} referenced in your client configuration.
+
+The file will be similar to the following:
+
+[source,java]
+----
+package com.hazelcast.namespaces.dyamicconfig;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.config.UserCodeNamespaceConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.map.EntryProcessor;
+
+public class Client {
+
+    public static void main(String[] args) {
+        HazelcastInstance client = HazelcastClient.newHazelcastClient();
+
+        EntryProcessor entryProcessor = new IncrementingEntryProcessor();
+        UserCodeNamespaceConfig namespaceConfig = new UserCodeNamespaceConfig("namespace_name"); <1>
+        namespaceConfig.addClass(YourEntryProcessor.class); <2>
```

Comment:
I assume here we want to demonstrate adding the `IncrementingEntryProcessor` class dynamically, right?

```suggestion
        namespaceConfig.addClass(IncrementingEntryProcessor.class); <2>
```